crates/node/src/network_server/service.rs

```diff
@@ -44,6 +45,9 @@ impl NetworkServer {
         // -- HTTP service (for prometheus et al.)
         let axum_router = axum::Router::new()
             .route("/metrics", get(render_metrics))
+            .route("/debug/pprof/heap", get(pprof::heap))
+            .route("/debug/pprof/heap/activate", get(pprof::activate_heap))
+            .route("/debug/pprof/heap/deactivate", get(pprof::deactivate_heap))
```

Comment:
Should we make those as POST?

Reply:
activate and deactivate yes it should probably be PUT

Reply:
PUT will be _very_ easy to forget :)

Reply:
one wonders if we should just accept any of them then :D